.github/prep-hotfix-sources.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -x
+
+HOTFIX_TAG=$1
+REPO_PATH=$2
+SPEC_PATH=$3
+
+# Create OSG build dirs
+build_dir=$(pwd)/_build_dir
```

Comment:
```suggestion
build_dir=_build_dir
```

Even if you want to use a variable for this, you don't really need to include `$PWD`, since you never `cd` in this script